packages/create-crc-app/templates/fec.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	appUrl: @@appUrl,
```

Comment:
Why different format than in `interceptChromeConfig: {{interceptChromeConfig}},` ?

Reply:
I think I had some reasons for it let me check

Reply:
@rvsia I remember now, the mustache is escaping some character to HTML symbols (eg: `&#39;`). Let me check if we can turn it off.